dynamo: Remove passing or deleted dynamo_expected_failures

c00w opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* #160151
* __->__ #159691

partially generated with
```
for TESTCASE in $(ls | cut -f1 -d'.' | grep -v CPython | uniq); do if grep "$TESTCASE" -m 1 .. -r; then echo; else   sl rm "$TESTCASE"* ; fi; done
```

c00w committed:
dynamo: Remove passing dynamo_expected_failures

[ghstack-poisoned]

c00w committed:
Update on "dynamo: Remove passing dynamo_expected_failures"




[ghstack-poisoned]

c00w committed:
Update on "dynamo: Remove passing dynamo_expected_failures"




[ghstack-poisoned]

c00w committed:
Update on "dynamo: Remove passing dynamo_expected_failures"




[ghstack-poisoned]

c00w committed:
Update on "dynamo: Remove passing dynamo_expected_failures"




[ghstack-poisoned]

c00w committed:
Update on "dynamo: Remove passing dynamo_expected_failures"


partially generated with
```
for TESTCASE in $(ls | cut -f1 -d'.' | grep -v CPython | uniq); do if grep "$TESTCASE" -m 1 .. -r; then echo; else   sl rm "$TESTCASE"* ; fi; done
```



[ghstack-poisoned]

c00w committed:
Update on "dynamo: Remove passing dynamo_expected_failures"


partially generated with
```
for TESTCASE in $(ls | cut -f1 -d'.' | grep -v CPython | uniq); do if grep "$TESTCASE" -m 1 .. -r; then echo; else   sl rm "$TESTCASE"* ; fi; done
```



[ghstack-poisoned]

c00w commented:
@pytorchbot merge

izaitsevfb commented:
@pytorchbot revert -m "breaking dynamo tests" -c nosignal

test_reductions.py::TestReductionsCPU::test_var_vs_numpy_cpu_float32 [GH job link](https://github.com/pytorch/pytorch/actions/runs/16816487731/job/47635319596) [HUD commit link](https://hud.pytorch.org/pytorch/pytorch/commit/36f46d082a4954921cb8493223f000f2aab79ed7)

c00w committed:
Update on "dynamo: Remove passing or deleted dynamo_expected_failures"


partially generated with
```
for TESTCASE in $(ls | cut -f1 -d'.' | grep -v CPython | uniq); do if grep "$TESTCASE" -m 1 .. -r; then echo; else   sl rm "$TESTCASE"* ; fi; done
```



[ghstack-poisoned]
